src/azure-cli/azure/cli/command_modules/util/custom.py

```diff
@@ -125,8 +125,9 @@ def upgrade_version(cmd, update_all=None, yes=None):  # pylint: disable=too-many
             logger.warning("Exit the container to pull latest image with 'docker pull mcr.microsoft.com/azure-cli' "
                            "or run 'pip install --upgrade azure-cli' in this container")
         elif installer == 'MSI':
+            from azure.cli.core.util import start_powershell_process
             logger.debug("Update azure cli with MSI from https://aka.ms/installazurecliwindows")
-            exit_code = subprocess.call(['powershell.exe', '-No-Profile', "Start-Process msiexec.exe -Wait -ArgumentList '/i https://aka.ms/installazurecliwindows'"])  # pylint: disable=line-too-long
+            exit_code = start_powershell_process("msiexec.exe -Wait -ArgumentList '/i https://aka.ms/installazurecliwindows'")  # pylint: disable=line-too-long
```

Comment:
[`-ArgumentList`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/start-process?view=powershell-7#parameters) is a parameter of `Start-Process`. Should we break it?